packages/shopping/src/controllers/user.controller.ts

```diff
@@ -83,6 +95,7 @@ export class UserController {
   }
 
   @get('/users/{userId}', {
+    security: SECURITY_SPEC_OPERATION,
```

Comment:
IIUC, this is configuring `basicAuth` schema. Is that intentional? I thought that we use `bearerAuth` schema everywhere. Or is this just for the spike?

Reply:
@bajtos Oh this is just a demo, for spike only, I just want to verify that a operation level security policy(which is different than the global `bearerAuth`) works.

>I thought that we use bearerAuth schema everywhere

Exactly 💯